django/contrib/postgres/forms/jsonb.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(forms.CharField):
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
+    }
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('widget', forms.Textarea)
+        super(JSONField, self).__init__(**kwargs)
+
+    def to_python(self, value):
+        try:
+            return json.loads(value)
+        except ValueError:
```

Comment:
This is the `to_python` of the form field, not the model field. It would under normal circumstances always receive a string. We do need to handle empty values though, I'll add that.